api/internal/conf/conf.go

```diff
@@ -100,9 +101,11 @@ type Config struct {
 	Authentication Authentication
 }
 
-// TODO: it is just for integration tests, we should call "InitLog" explicitly when remove all handler's integration tests
+// ENV=test is for integration tests only, other ENV should call "InitConf" explicitly
```

Comment:
TODO should be retained here. We need to finally remove `init()`.

Reply:
> TODO should be retained here. We need to finally remove `init()`.

@nic-chen Done, thanks.